klaus/contrib/wsgi_autoreload.py

```diff
@@ -49,18 +50,20 @@ def app(environ, start_response):
     return app
 
 
+if 'KLAUS_REPOS' in os.environ:
+    warnings.warn("use KLAUS_REPOS_ROOT instead of KLAUS_REPOS for the autoreloader apps", DeprecationWarning)
+
 if 'KLAUS_HTDIGEST_FILE' in os.environ:
     with open(os.environ['KLAUS_HTDIGEST_FILE']) as file:
-        application = make_app(
-            os.environ['KLAUS_REPOS'],
+        application = make_autoreloading_app(
```

Comment:
Thanks for fixing this, this was an oversight from 665a7292ab68fecbc337d116a3d91598bf7714de